### PR TITLE
AICORE-460: alignment rest api access

### PR DESCRIPTION
**Pull request by vpasquier**

### CODE REVIEW

**Comment by vpasquier** [comment]
@jcarsique i need a review on the pom of the internal - lets see that in a meeting

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <groupId>org.nuxeo.ecm.distribution</groupId>
     <artifactId>nuxeo-distribution</artifactId>
     <!-- greater or equal to runtime HF required version -->
-    <version>10.10-HF39</version>
+    <version>10.10-HF44</version>
```
Comment (jcarsique) [inline pom.xml:8]: Upgrade this version only if needed for compile or runtime: the version stated here is used as a minimal requirement for the Marketplace packages.

File: pom.xml [hunk 2]
```diff
@@ -146,6 +147,21 @@
         <classifier>tests</classifier>
         <version>${nuxeo.platform.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-url</artifactId>
+        <version>10.10</version>
```
Comment (jcarsique) [inline pom.xml:153]: <version>${nuxeo.platform.version}</version>

File: pom.xml [hunk 3]
```diff
@@ -146,6 +147,21 @@
...
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-types</artifactId>
+        <version>10.10</version>
```
Comment (jcarsique) [inline pom.xml:158]: <version>${nuxeo.platform.version}</version>

File: pom.xml [hunk 4]
```diff
@@ -146,6 +147,21 @@
...
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-tag</artifactId>
+        <version>10.10</version>
```
Comment (jcarsique) [inline pom.xml:163]: <version>${nuxeo.platform.version}</version>

File: pom.xml [hunk 5]
```diff
@@ -36,6 +36,7 @@
     <module>nuxeo-ai-model</module>
     <module>nuxeo-ai-config</module>
     <module>nuxeo-ai-web-ui</module>
+    <module>nuxeo-ai-internal</module>
```
Comment (jcarsique) [inline pom.xml:39]: This module no more needs a special treatment, remove the dedicated lines in tools/release.sh and tools/sprint.sh
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/release.sh#L43
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/release.sh#L58
https://github.com/nuxeo/nuxeo-ai/blob/766ddbd25b9f13f58c0c7ba931eea6440218b1f7/tools/sprint.sh#L128

**Review by jcarsique — CHANGES_REQUESTED** [review]

**Review by andreinechaev — APPROVED** [review]

**Pull request merged**